include/mlir/Dialect/StandardOps/Ops.td

```diff
@@ -950,6 +966,10 @@ def SelectOp : Std_Op<"select", [NoSideEffect, SameOperandsAndResultShape]> {
   let hasFolder = 1;
 }
 
+def SignOp : FloatUnaryOp<"sign"> {
```

Comment:
SignFOp, as this is limited to floats and a SignIOp would make sense. If it is SignOp, I would prefer it to work on both.